apps-rendering/riff-raff.yaml

```diff
@@ -1,20 +1,32 @@
 stacks: [mobile, mobile-preview]
 regions: [eu-west-1]
-deployments:
-  mobile-apps-rendering-cfn:
-    type: cloud-formation
-    app: mobile-apps-rendering
-    parameters:
+templates:
+  cloudformation:
+     type: cloud-formation
```

Comment:
Are these new spaces valid? Should we ident by 2 rather than 3?

Reply:
We moved the statement to the correct indentation (2).